BrainPortal/lib/cbrain_extensions/string_extensions/utilities.rb

```diff
@@ -34,7 +34,7 @@ module Utilities
       #
       #   puts "".bash_escape                     => ''
       #   puts "abcd".bash_escape                 => abcd
-      #   puts "Mike O'Connor".bash_escape        => 'Mike O'\'Connor
+      #   puts "Mike O'Connor".bash_escape        => 'Mike O'\'Connor'
```

Comment:
Don't introduce errors in the documentation. The comment was correctL there is no final single quote. Try it in the console.